tools/dev/run_dev_server.sh

```diff
@@ -63,8 +63,8 @@ case $CONFIG in
       CLUSTER_DATA_BIND_PORT="7103" \
       CLUSTER_JOIN="localhost:7100" \
       CONTEXTIONARY_URL=localhost:9999 \
-      DEFAULT_VECTORIZER_MODULE=text2vec-contextionary \
-      ENABLE_MODULES="text2vec-contextionary,backup-filesystem" \
+      DEFAULT_VECTORIZER_MODULE=none \
+      ENABLE_MODULES="backup-filesystem" \
```

Comment:
Maybe we can add an additional profile, like `second-node-no-modules` or something, as this profile was designed to work with `local-development`